### PR TITLE
Remove nerdctl (post v92)

### PR DESCRIPTION
**Pull request by mike-sul**
Depends on https://github.com/foundriesio/meta-lmp/pull/1369 and https://github.com/foundriesio/aktualizr-lite/pull/290.

### CODE REVIEW

File: meta-lmp-base/recipes-containers/composeapp/composectl_git.bb [hunk 1]
```diff
@@ -19,7 +19,7 @@ GO_EXTRA_LDFLAGS = "\
...
 "
 
-#do_install:append() {
-# TODO
-#}
-
+do_install:append() {
+    cd ${D}/${bindir}
+    ln -sf composectl aklite-apps
+}
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-containers/composeapp/composectl_git.bb:25]: Add this change as part of the "base: rc: Add composectl recipe" commit.

**Comment by ricardosalveti** [comment]
Please rebase.

File: meta-lmp-base/recipes-containers/composeapp/composectl_git.bb (hunk 1 above)
Comment (ricardosalveti) [reply meta-lmp-base/recipes-containers/composeapp/composectl_git.bb:25]: Ignore, part of https://github.com/foundriesio/meta-lmp/pull/1369.

**Comment by mike-sul** [comment]
@ricardosalveti This one is ready to go now.

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**